### PR TITLE
only test available date range on 32-bit

### PR DESCRIPTION
**Pull request by remicollet**
Related to #1347

### CODE REVIEW

File: test/Faker/Provider/fi_FI/PersonTest.php [hunk 1]
```diff
@@ -44,18 +44,25 @@ public function testPersonalIdentityNumberUsesBirthDateIfProvided($seed, $birthd
...
+        if (strtotime('1800-01-01 00:00:00')) {
+            $min="1900";
+            $max="2099";
+            for ($i = 0; $i < 10; $i++) {
+                $birthdate = $this->faker->dateTimeBetween('1800-01-01 00:00:00', '1899-12-31 23:59:59');
+                $pin = $this->faker->personalIdentityNumber($birthdate, NULL, true);
+                $this->assertRegExp('/^[0-9]{6}\+[0-9]{3}[0-9ABCDEFHJKLMNPRSTUVWXY]$/', $pin);
+            }
+        } else { // timestamp limit for 32-bit computer
+            $min="1902";
```
Comment (fzaninotto) [inline test/Faker/Provider/fi_FI/PersonTest.php]: why not 1900?

Comment (remicollet) [reply test/Faker/Provider/fi_FI/PersonTest.php]: because strtotime("1900...") return false on 32-bit

Comment (remicollet) [reply test/Faker/Provider/fi_FI/PersonTest.php]: # php -r 'var_dump(strtotime("1900-01-01"));'
bool(false)
# php -r 'var_dump(strtotime("1902-01-01"));'
int(-2145916800)
# php -r 'var_dump(strtotime("2038-01-01"));'
int(2145916800)
# php -r 'var_dump(strtotime("2039-01-01"));'
bool(false)

Comment (fzaninotto) [reply test/Faker/Provider/fi_FI/PersonTest.php]: Understood. According to http://php.net/manual/en/function.strtotime.php, the safe limits are 1902 and 2037. So why 2036 in your code?

Comment (remicollet) [reply test/Faker/Provider/fi_FI/PersonTest.php]: 2036 was a typo ;)

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**